plasma_framework/contracts/src/exits/payment/PaymentExitDataModel.sol

```diff
@@ -23,7 +23,6 @@ library PaymentExitDataModel {
 
     struct InFlightExit {
         uint256 exitStartTimestamp;
-        uint256 exitPriority;
```

Comment:
sorry for this, as I pointed out in another comment, though we don't need `priority` but we would need to save `position` to derive the `priority` when enqueue.

Reply:
done